docs/رموز_الأخطاء/م/م٠٠٠١.md

```
@@ -0,0 +1,99 @@
+
+> **ملاحظة**: حالياً لا توجد كلمات مفتاحية مهملة في ترقيم v1.0. هذا الرمز مُعدّ للاستخدام المستقبلي عندما تُستبدل بعض الصيغ بصيغ أحدث.
+
+## مثال افتراضي
+
+```tarqeem
+// مثال افتراضي - ليس حقيقياً حالياً
+كلمة_قديمة س = 5    // ⚠️ تحذير: 'كلمة_قديمة' مهملة، استخدم 'متغير'
+```
+
+**الرسالة**:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)